Monika After Story/game/script-topics.rpy

```diff
@@ -2790,6 +2790,12 @@ label monika_love:
         $ persistent._mas_monika_lovecountertime = datetime.datetime.now()
         return
 
+    #Event for when Monika tells the player she loves him.
+    #Triggered if the player immediatelly reacts with "I love you".
+    elif persistent._mas_monika_love_reaction_event != 0:
+        call monika_love_reaction_event
```

Comment:
You're only using this dialogue once, shouldn't really need a label for it.

Reply:
Since this can potentially get really long, I did not want it to be with the other code.